src/lib/rule-finder.js

```diff
@@ -23,7 +23,7 @@ function _getConfig(configFile) {
     // Point to the particular config
     configFile
   });
-  return cliEngine.getConfigForFile();
```

Comment:
doesn't this only return the proper config for the config file, which might be yaml or `.eslintrc` json?

Config files are often not JS.

Reply:
Perhaps. Consider this more of a bug report than a PR then. I'm not sure what the proper solution for this is. Because for this project there's not really a specific file that we're looking for a config for 🤔

Reply:
#304 is the bug report, which includes my latest attempt: https://github.com/sarbbottam/eslint-find-rules/compare/master...ljharb:eslint_6

Reply:
No worries, to be fair I was super excited when I saw the PR notification because this package's v6 support is blocking me in two shared configs :-/

Reply:
@ljharb (or anyone else looking for issues related to `.js` configs as I was and finding this comment)
re:
> Config files often not JS.

by the way:
If you look at the top eslint configs on the first page of NPM, 12 out of the 18 use the `.js` config style (as opposed to `.json` or `.yaml`).  The list of popular configs using this style includes:
- [eslint-config-airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/index.js)
- [eslint-config-alloy](https://github.com/AlloyTeam/eslint-config-alloy/blob/master/index.js)
- [eslint-config-amo](https://github.com/mozilla/eslint-config-amo/blob/master/index.js)
- [eslint-config-canonical](https://github.com/gajus/eslint-config-canonical/blob/master/eslintrc.js)
- [eslint-config-create-react-app](https://github.com/facebook/create-react-app/blob/master/packages/eslint-config-react-app/index.js)
- [eslint-config-expo](https://github.com/expo/expo/blob/master/.eslintrc.js)
- [eslint-config-fbjs](https://github.com/facebook/fbjs/blob/master/packages/eslint-config-fbjs/index.js)
- [eslint-config-google](https://github.com/google/eslint-config-google/blob/master/index.js)
- [eslint-config-peerigon](https://github.com/peerigon/eslint-config-peerigon/blob/master/es5.js)
- [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier/blob/master/index.js)
- [eslint-config-synacor](https://github.com/synacor/eslint-config-synacor/blob/master/src/index.js)
- [eslint-config-vi](https://github.com/vicompany/eslint-config-vi/blob/master/config/node.js)
- [eslint-config-vue](https://github.com/vuejs/eslint-plugin-vue/blob/master/.eslintrc.js)

(I already had this list)

Reply:
sure, but these are the top *shared configs*. I have 200 packages that only use the rc form, and a shared config for many of them that also does. Config files are indeed often not JS, even though they are also often JS :-)